### PR TITLE
add p-nand-q gtools

### PR DESCRIPTION
**Pull request by raisercostin**

### CODE REVIEW

File: gtools.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        "pserv4.exe",
+        "pserv4.vshost.exe",
+        "regdiff.exe",
+        "regdiff.vshost.exe",
+        "remove_admin_shares.exe",
+        "su.exe",
+        "touch.exe",
+        "which.exe"
+    ],
+    "checkver": {
```
Comment (Ash258) [inline gtools.json]: `"checkver": "<h1>gtools ([\\d.]+)<"` Should work too

File: gtools.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "http://p-nand-q.com/download/gtools",
+    "version": "4.2",
+    "description": "Gtools include: pserv, dllusage, gkalk, as well as a couple of command-line only tools: regdiff, supershell, ddupe, pathed, touch, which, md5sums.",
+    "license": "New BSD License",
+    "hash": "sha1:7152656b8698daf80027861a2bedba236a3185df",
+    "url": "http://p-nand-q.com/download/gtools/gtools-4.2.exe#dl.7z",
+    "bin": [
```
Comment (Ash258) [inline gtools.json]: Add `"post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\" -Recurse",` to delete junk folder with installer files

**Comment by r15ch13** [comment]
`.vhost.exe` are .net files that are possibly not needed.

**Comment by rasa** [comment]
Sorry for the delay.

**Pull request merged**